dietpi/dietpi-software

```diff
@@ -3617,6 +3642,22 @@ Package: openssl libssl*\nPin: origin packages.sury.org\nPin-Priority: -1' > /et
 
 		fi
 
+		software_id=125 # Synapse Server
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+			Banner_Installing
+
+			# Install dependencies for running on ARMv6/7 (piwheels) or building
```

Comment:
There is an `x86_64` wheel as well, at least I think it's one: https://pypi.org/project/matrix-synapse/#files
I'll test, just leave the note to not forget.

Reply:
What are libopenjp2-7 libtiff5 libxcb1 for? I have none of them installed on x86_64 and build + service startup + web access work fine. Only psycopg2 needs to be compiled there (libpq-dev + gcc required), but matrix-synapse is pre-compiled.

Reply:
Whenever I tried to run it on RPi 32-bit, it gave me import errors until I installed those packages.  Strange, since none of them are listed on their piwheels project page.